search/lib/convert/granules.js

```diff
@@ -255,6 +259,10 @@ function cmrGranulesToFeatureCollection (event, cmrGrans) {
     });
   }
 
+  if (granulesResponse.features.length < 10) {
+    granulesResponse.links.splice(granulesResponse.links.length - 1);
+  }
```

Comment:
I wonder if we want to extract/abstract this pagination behavior somehow so it can be shared by all of the apis that use it.

Reply:
This is a good idea and something I'd like to do if we have time.

Reply:
But yeah, looks good for now.

Might be nice to extract it so we can do all of the unit testing of that behavior in one place. As, in my experience, pagination can be error-prone.